### PR TITLE
[OP-55] Add flag that enables auto photo verification

### PR DESCRIPTION
**Pull request by vladhrech**
**Description:** 
Add flag that enables auto photo verification on registration

**Youtrack:** 
https://youtrack.raccoongang.com/issue/OP-55

- [x] Demo is shown to QA

### CODE REVIEW

File: common/djangoapps/student/views/management.py [hunk 1]
```diff
@@ -741,6 +742,13 @@ def create_account_with_params(request, params):
     else:
         compose_and_send_activation_email(user, profile, registration)
 
+    if settings.FEATURES.get('AUTO_SOFTWARE_SECURE_PHOTO_VERIFICATION'):
+        SoftwareSecurePhotoVerification.objects.create(
+            status="approved",
+            user_id=user.id,
+            submitted_at=datetime.datetime.now(UTC),
+            photo_id_key=uuid.uuid4())
```
Comment (arsentur) [inline common/djangoapps/student/views/management.py:750]: Please, move the trailing bracket to the next line.

File: common/djangoapps/student/views/management.py [hunk 2]
```diff
@@ -741,6 +742,13 @@ def create_account_with_params(request, params):
     else:
         compose_and_send_activation_email(user, profile, registration)
 
+    if settings.FEATURES.get('AUTO_SOFTWARE_SECURE_PHOTO_VERIFICATION'):
```
Comment (arsentur) [inline common/djangoapps/student/views/management.py:745]: Please, add comments for the new code.

File: common/djangoapps/student/views/management.py (hunk 1 above)
Comment (arsentur) [inline common/djangoapps/student/views/management.py:750]: It is ok to fill in `uuid.uuid4()`? Can this affect another modules or something else?

Comment (vladhrech) [reply common/djangoapps/student/views/management.py:750]: Will be changed in next commit

File: common/djangoapps/student/views/management.py (hunk 2 above)
Comment (vladhrech) [reply common/djangoapps/student/views/management.py:745]: Will be added in next commit

File: common/djangoapps/student/views/management.py (hunk 1 above)
Comment (vladhrech) [reply common/djangoapps/student/views/management.py:750]: You're right, I will change it to the format that is used on photo verification

**Pull request merged**